_posts/2018-09-24-galaxy-hts-data-analysis-workshop.md

```diff
@@ -0,0 +1,22 @@
+---
+site: freiburg
+title: A successuful Galaxy HTS data analysis workshop
+location: Freiburg, Germany
+tags: [training]
+---
+
+<div class="multiple-img">
+<a data-flickr-embed="true" data-header="true" data-footer="true"  href="https://www.flickr.com/photos/134305289@N03/43888095455/in/datetaken/" title="Galaxy HTS data analysis workshop @ Freiburg"><img src="https://farm2.staticflickr.com/1892/43888095455_3238320d36_z.jpg" width="640" height="427" alt="Galaxy HTS data analysis workshop @ Freiburg"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
+</div>
+
+A new Galaxy workshop on HTS data analysis was held in Freiburg last week. During this week, the [Freiburg Galaxy Team](https://usegalaxy.eu/freiburg/) and [Bionformatics facility from the MPI-IE Freiburg](https://www.ie-freiburg.mpg.de/bioinformaticsfac) offered an introductory course to the analysis of high-throughput sequencing data using Galaxy:
+
+- Monday 17th: HTS and Galaxy presentations and introduction by Thomas Manke and Björn Grüning, with the help of Mehmet Tekman
+- Tuesday 18th: Quality control, Mapping, ChIP-seq analysis by Bérénice Batut and Leily Rabbani, with the help of Devon Ryan
+- Wednesday 19th: RNA-seq analysis by Anika Erxleben and Bérénice Batut, with the help of Pavan Videm and Mehmet Tekman
+- Thursday 20nd: HiC data analysis by Ralf Gilsbach, Devon Ryan and Joachim Wolff
+- Friday 21st: Exercises & discussion
+
+2 Master students, 16 PhD students, 5 Post-docs and 2 PIs participated to this workshop. Their feedbacks were good: 8% considered the course as good, 54% as very good and 39% excellent, all participants will recommend it and 96% will use Galaxy for future projects. A quite sucessful week then!
```

Comment:
This was the feedback we received - instead of "Their feedbacks were..."

Also do we want to include this information?

Reply:
I though that he feedback is good so why not mentioning it and showing that people seem to like the workshop :smile:
Do you think we should remove this part?

Reply:
Feel free to ignore this but if it was me I would add up those percentages so that the first one you see is not 8% e.g. 101% of people though it was good to excellent (those numbers seem to add up to 101%? 8+54+39)